### ISSUE TITLE
Update AirConnect to 1.x.x

### ISSUE THREAD

**Issue by eizedev** [eizedev/AirConnect-Synology#57]
## Is your feature request related to a problem? Please describe

Nope

## Describe the solution you'd like

A fully refactored version of AirConnect was released. Testing and packaging is needed.  
Update to 1.0.1 or newer version will be planed.

## Describe alternatives you've considered

None

## Additional context

https://github.com/philippe44/AirConnect/releases

**Comment by alekslyse** [comment]
Yes, please this is actually important, as synology refuse to update its OS with the current AirConnect package

**Comment by eizedev** [comment]
@alekslyse Which version of DSM are you using? I have updated to the latest version without any problems resulted by AirConnect-Synology. But you need to reinstall AirConnect-Synology after the upgrade since Synology will remove it during upgrade in my case.
